src/burn/engine/plan.cpp

```diff
@@ -1920,6 +1927,10 @@ static HRESULT GetActionDefaultRequestState(
 
     switch (action)
     {
+    case BOOTSTRAPPER_ACTION_CACHE:
+        *pRequestState = BOOTSTRAPPER_REQUEST_STATE_CACHE;
```

Comment:
Good point. I'll need to go back and check our internal code since this has changed a bit to see if there's a problem there. Looking through the current and request states, seems it's just (current->requested): present->present, cache->none (though I can't find anything in the code *not commented out* that sets BOOTSTRAPPER_PACKAGE_STATE_CACHE), and *->cache.